CoreScriptsRoot/ServerCoreScripts/ServerSocialScript.lua

```diff
@@ -38,7 +38,6 @@ local function decodeJSON(json)
 		return HttpService:JSONDecode(json)
 	end)
 	if not success then
-		print("decodeJSON() failed because", result, "Input:", json)
```

Comment:
I have only seen it in studio mode. Maybe we could use a :IsStudio() and return if true, and remove the other two. As it is confusing especially when using printing something similar when sending HTTP. Or would you just like to leave alone?